srv/salt/ceph/stage/prep/master/default-update-reboot.sls

```diff
@@ -1,4 +1,3 @@
-
```

Comment:
this initial newline is probably there for a reason - it's in all the other files, so reinstating it